src/main/java/com/minecolonies/blockout/PaneParams.java

```diff
@@ -415,7 +416,7 @@ public int getColorAttribute(final String name, final int def)
 
         final Matcher m = RGBA_PATTERN.matcher(attr);
 
-        if (attr.startsWith("#"))
+        if (attr.charAt(0) == HASH_CHAR)
         {
             //  CSS Hex format: #00112233
             return Integer.parseInt(attr.substring(1), 16);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 16 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)